assignments/lambda-classes.js

```diff
@@ -1 +1,145 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attributes) {
+        this.newObjectName = attributes.name;
```

Comment:
totally legit to name it `newObjectName` but may be a bit easier to read if it's just `name` etc 🤷‍♀️